File: php/gen.php
<?php


$data = file_get_contents("../set.txt");
$data = explode("\n",$data);
$i = 0;


$start = microtime(true);
foreach($data as $v){
    if (strpos($v, "abcde") !== false){
        $i++;
    }
}

echo(sprintf("PHP in %dms - %d\n",(microtime(true) - $start)*1000,$i));

File: lua/gen.lua
-- LuaJIT LangBench impl

local i = 0
local file = io.open("../set.txt", "r");
local data = {}
for line in file:lines() do
    table.insert (data, line);
end

local start = os.clock()
for idx = 1, #data, 1 do
    if string.find(data[idx], "abcde") then
        i = i + 1
    end
end

print(string.format("LuaJIT in %dms - %d", (os.clock() - start) * 1000, i))